lib/libwebsockets.h

```diff
@@ -234,15 +234,7 @@ struct libwebsocket_pollargs {
     int prev_events;   // the previous event mask
 };
 
-#ifdef _WIN32
```

Comment:
This is to avoid trouble like this:

w:/libwebsockets/test-server/test-server.c: In function 'main':
w:/libwebsockets/test-server/test-server.c:965:11: warning: passing argument 2 of 'libwebsocket_service_fd' from incompatible pointer type [enabled by default]
           &pollfds[n]) < 0)
           ^
In file included from w:/libwebsockets/test-server/test-server.c:44:0:
w:/libwebsockets/test-server/../lib/libwebsockets.h:1165:1: note: expected 'struct libwebsocket_pollfd *' but argument is of type 'struct pollfd *'
 libwebsocket_service_fd(struct libwebsocket_context *context,
 ^

resulting from the fact that pollfd is already defined in winsock2.h included by libwebsockets.h. Seems not to be anything specific to MinGW:

https://msdn.microsoft.com/en-us/library/windows/desktop/ms740094(v=vs.85).aspx